deeppavlov/configs/odqa/en_odqa_infer_wiki.json

```diff
@@ -55,9 +63,17 @@
       "server_utils": "ODQA"
     },
     "download": [
+      "http://files.deeppavlov.ai/deeppavlov_data/squad_model_1.3_cpu_compatible.tar.gz",
```

Comment:
When `"config_path"` parameter is used, download links are found recursively, so this link would be found in `squad.json` and is not needed here.